libraries/Ticker/Ticker.h

```diff
@@ -129,6 +129,7 @@ class Ticker
 
 protected:
 	ETSTimer* _timer;
+	char _etsTimerMemory[sizeof(ETSTimer)];
```

Comment:
Please replace this with `ETSTimer _timerObj` or similar. Then do _timer = &_timerObj in attach and _timer = nullptr on detach. Remove new/delete.

Reply:
@devyte Naturally I don't have a hard time adopting your change, I hope it's OK that I resist it, given the explanations I made. As a compromise, if one's needed, I'd like to keep the placement syntax as comments in the code and otherwise I'll commit the change you ask for.